buildman/templates/dockersystemd.json

```diff
@@ -37,6 +55,7 @@ ExecStartPost={{ start_post }}
 {% if not oneshot -%}
 ExecStop=/usr/bin/docker stop {{ name }}
 {%- endif %}
```

Comment:
should this be just `%` instead of `%-`?

Reply:
Shouldn't matter. The dash just removes whitespace before that block.